.github/CODEOWNERS

```diff
@@ -272,9 +272,9 @@
 /comp/core/workloadmeta @DataDog/container-platform
 /comp/metadata/packagesigning @DataDog/agent-delivery
 /comp/trace/etwtracer @DataDog/windows-agent
+/comp/serializer/compression @DataDog/agent-metrics-logs
```

Comment:
I would assume we modified something in the logic that moved this component. Is that expected?

Reply:
Tricky to track down, but it was due to components being sorted by component name rather than path. Fixed now.